docs/csharp/tour-of-csharp/expressions.md

```diff
@@ -78,13 +78,13 @@ The following summarizes C#’s operators, listing the operator categories in or
 * Logical XOR
     - `x ^ y`: Integer bitwise XOR, boolean logical XOR
 * Logical OR
-    `x | y`: Integer bitwise OR, boolean logical OR
+    - `x | y`: Integer bitwise OR, boolean logical OR
 * Conditional AND
     - `x && y`: Evaluates `y` only if `x` is not `false`
 * Conditional OR
     - `x || y`: Evaluates `y` only if `x` is not `true`
 * Null coalescing
-    - `X ?? y`: Evaluates to `y` if `x` is null, to `x` otherwise
+    - `X ?? y`: Evaluates to `y` if `x` is `null`, to `x` otherwise
```

Comment:
Will change it back to null. But `y` and `x` are not keywords, so why are these formatted?